src/Sulu/Bundle/AudienceTargetingBundle/Tests/app/Resources/webspaces/sulu.io.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You only need that file to get a webspace returned, right? Then I would keep the file simpler (only one localization, one URL, one env, ...)